bin/easystar-0.4.4.min.js

```diff
@@ -0,0 +1,24 @@
+/*!
```

Comment:
Can we explore why comments aren't being stripped?

Also lets continue to generate a non-minified file unless we want to bump major versions.

Reply:
> Can we explore why comments aren't being stripped?

Comments are being actively stripped from the webpack build, we only keep the license banner.
We could alternatively use a comment like `@license is MIT, available at https://github.com/prettymuchbryce/easystarjs/blob/master/LICENSE`

> Also lets continue to generate a non-minified file unless we want to bump major versions.

I pushed https://github.com/prettymuchbryce/easystarjs/pull/82/commits/f359f194fb73a47287fe2351e82fe6ebf182b394
I've overlooked that previously the build included both minified/unminified versions inside the `bin` folder.

Take into consideration that having two versions doesn't bring any advantages.
The "unminified" version is wrapped in webpack code.
We could output `easystar.js` and `easystar.dev.js` which would have `sourceMap` attached to it.
This way we get rid of the version in the filename, and the scope of each file is much more clear.